File: Programmers/Lv0/배열비교하기/solution.py
def solution(arr1, arr2):
    arr1_size = len(arr1)
    arr2_size = len(arr2)

    if arr1_size > arr2_size:
        return 1
    elif arr2_size > arr1_size:
        return -1
    else:
        arr1_sum = sum(arr1)
        arr2_sum = sum(arr2)

        if arr1_sum > arr2_sum:
            return 1
        elif arr2_sum > arr1_sum:
            return -1

    return 0

File: Programmers/Lv0/배열비교하기/Solution.java
package Lv0.배열비교하기;

import java.util.Arrays;

class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int arr1Size = arr1.length;
        int arr2Size = arr2.length;

        if (arr1Size > arr2Size) return 1;
        else if (arr2Size > arr1Size) return -1;
        else {
            int arr1Sum = Arrays.stream(arr1).sum();
            int arr2Sum = Arrays.stream(arr2).sum();

            if (arr1Sum > arr2Sum) return 1;
            else if (arr2Sum > arr1Sum) return -1;
        }

        return 0;
    }
}